src/main/resources/functionDocs/get_tabcomplete_prototype

```diff
@@ -3,35 +3,48 @@ which provides an easy to use syntax for simple to moderately complex tab comple
 random strings, or a number.
 
-ClassTypes must be enum values, and will autocomplete with the list of values in that enum.
+==== ClassTypes ====
 
-Arrays are simply returned as is.
+Must be enum values, and will autocomplete with the list of values in that enum.
+* e.g. <code>WorldEnvironment</code>, which at time of writing is an enum containing the values {NORMAL, NETHER, THE_END}
 
-Closures are passed in the same values as the tabcompleter, and are expected to return an array. This is useful when
+==== Arrays ====
+
+Arrays are simply returned as is:
+* e.g. <code>array(1, 2, 3)</code>
+
+==== Closures ====
+
+You can pass a closure the same parameters as the tabcompleter, and they are expected to return an array. This is useful when
 the completion logic varies based on the arguments passed in, such as the current player executing the tabcompleter.
 
-Associative arrays are used when you need to provide dynamic functionality based on previous arguments. Only enum or
-array values are supported as "previous" arguments.
-The keys of the array are used to determine which value is selected, and the values of
-the array are one of the above completion types. The keys follow the general format of "<enumValue" where "<" means that
-we're comparing to the previous value, and "enumValue" is the selector that was entered by the user. "<" can also point
-to more than one previous value, so for instance "<<enumValue" would also work, meaning compare to the argument provided
-two prior. If you have multiple selectors that have the same handler, they can be combined into one pipe separated
-string, for instance "<enumValue1|<enumValue2". Given that you can mix multiple argument selector types, it's possible
-you have a conflict, in this case, the behavior is undefined, so it's recommended that you don't mix argument depth
-indicators (don't use < and << in the same group). Using "<" as a key is the default value, and is what is used if
-nothing else matches. In case no keys match, then no completions will be provided.
-
-In all cases, once the list of completions has been determined, it is compared against the existing input so far
-of the user, and only matches that start with the current input are returned.
-
-Calling get_tabcomplete_prototype with no arguments disables tab completion (always returns an empty array).
\ No newline at end of file
+==== Associative arrays ====
+
+These are used when you need to provide dynamic functionality based on previous arguments. Only enum or array values are supported as previous arguments.
+The keys of the array:
+* are used to determine which parameter to look at, and what selector value that parameter must have.
+* follow the general format of <code>&lt;enumValue</code>, where <code>&lt;</code> means that we're comparing to parameter immediately before this one, and <code>enumValue</code> is the selector value that was entered by the user into that parameter.
```

Comment:
Put newlines in this, to keep to the roughly 120 column width. For bullet points, since the newline means something, and it will mess up the formatting, end the line with `\`, and the parser will remove the newline before applying the wiki format. (This is true in any line, but it only really matters with bullet points.)

Same thing for the bullet points below as well.